context/app/static/js/components/detailPage/Protocol/Protocol.jsx

```diff
@@ -3,38 +3,40 @@ import PropTypes from 'prop-types';
 import Divider from '@material-ui/core/Divider';
 
 import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
-import useProtocolData from 'js/hooks/useProtocolData';
+import useProtocolData, { useFormattedProtocolUrls } from 'js/hooks/useProtocolData';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import { StyledPaper } from './style';
 import SectionItem from '../SectionItem';
 
-function ProtocolLink({ title, resolverHostnameAndDOI }) {
+function ProtocolLink({ url }) {
+  const { loading, data, error } = useProtocolData(url);
+  if (error) {
+    <SectionItem label="Error loading protocol info">
+      This protocol may be private or otherwise inaccessible.
+    </SectionItem>;
+  }
+  if (loading || !data) {
+    <SectionItem label="Loading protocol info">Please wait...</SectionItem>;
```

Comment:
@tsliaw do you want to weigh in on the language used here?

Reply:
@NickAkhmetov Yes that language should be changed. Please change to "Protocols are loading. If protocols take a significant time to load, please contact help@hubmapconsortium.org about this issue and mention the HuBMAP ID." And add the email icon after the email. The styling of that loader might need to change in the future since it doesn't match the rest of the portal, but this should be sufficient for now.

Reply:
I've modified the text accordingly:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/dbdb6a46-2124-4c7e-90ac-05b9518ff6c0)

I've also added handling to make sure that the loading message is only displayed once in cases where there's more than one protocol URL.